wiki/FAQ.md

```diff
@@ -81,10 +81,8 @@
 
 
   ## How to create an order with takeProfit+stopLoss?
-  To create an order with both takeProfit and stopLoss parameters, you'll need to use a method that supports setting these parameters directly if the exchange allows it. You can check by looking at `exchange.has['createOrderWithTakeProfitAndStopLoss']`, `exchange.has['createStopLossOrder']` and `exchange.has['createTakeProfitOrder']`.
```

Comment:
obsolete approach, and I doubt in reality advanced users would be using this approach (they would have their internal custom meta, because this approach was very fragile and hard to follow)